spec/definitions/Taxes.yaml

```diff
@@ -0,0 +1,14 @@
+type: object
+description: A configuration for a tax.
+required:
+  - name
+  - value
+properties:
+  name:
```

Comment:
`label`, I would recommend.

Reply:
but this is a tax name there, not a just some label
maybe i'm wrong

Reply:
it's the label that appears:

`GST:`  $100  (looks like a label to the end user)